### ISSUE TITLE
This isnt an issue but!

### ISSUE THREAD

**Issue by RBXLModders** [Senfarius/Penguin-Chat-2#1]
This is not unofficial!!!! you can even see the original creator even stated that gratis-ting.dk was a part of penguin chat. Do you know how rare this is? This is WAYYYYY OVER EXTREME rare. https://web.archive.org/web/20030615200648if_/http://www.rocketsnail.com:80/chat/penguin/index.html is a link to the statement from 2003.

**Comment by Senfarius** [comment]
Now I see that you may be right. I used the decompiler and there is indeed a logo of gratis-ting. At the time I wasn't sure whether it was the original version or not, because there were a lot of "remakes" back then (for example a huge amount of remakes on rile5 and other forums). In the case of this release, I am still not entirely sure if these are actually the original files by Rocketsnail or if they are somehow recreated by someone else and based on the "gratis-ting" version.

**Comment by RBXLModders** [comment]
is there possibly any way you can translate artwork.swf to english?

**Comment by Senfarius** [comment]
Not really. I can decompile the file, but I don't have any software to edit this.
